src/components/views/messages/MPollBody.tsx

```diff
@@ -171,15 +163,18 @@ export default class MPollBody extends React.Component<IBodyProps, IState> {
         }
 
         const pollId = this.props.mxEvent.getId();
-        const votes = this.collectVotes();
+        const userVotes = this.collectUserVotes();
+        const votes = countVotes(userVotes, this.props.mxEvent.getContent());
         const totalVotes = this.totalVotes(votes);
+        const userId = MatrixClientPeg.get().getUserId();
```

Comment:
can we use a MatrixClientContext here instead of MatrixClientPeg?

Reply:
Something like 17b726cfb0 ?  I've never used this before so please check I did it right :-)

Reply:
Yup! This way will make any future transitions to a multi-account world much easier, thanks!